### PR TITLE
Implement Add car functionality

### PR DESCRIPTION
**Pull request by GordonOtieno**
- Created add car form.
- Styled add car form.
- Connected the form component to the api.
- Post data to the api.
- solved linters errors.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -126,3 +155,58 @@ div.mycard .myrow {
...
+.form-container {
+  background: var(--light);
+  backdrop-filter: blur(3rem);
+  filter: drop-shadow(0 0 10px rgba(0, 0, 0, 0.2));
+  padding: 1.5rem 5rem;
+  border-radius: 15px;
+  justify-content: center;
+  width: 100%;
+  max-width: 800px;
+}
```
Comment (apt2tell) [inline src/App.css:168]: > consider adding some margin-top to the form container to allow for viewing the first field

img_blob

File: src/components/addcar/AddCar.jsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+          />
+          <FileUpload
+            type="upload"
+            file={file}
+            preview={preview}
+            handleFileChange={(input) => handleFileChange(input)}
+          />
+
+          <button type="submit">Add Bike</button>
+        </form>
```
Comment (apt2tell) [inline src/components/addcar/AddCar.jsx:117]: > consider also changing the button text to Add Car instead

img_blob2

**Review by apt2tell — CHANGES_REQUESTED** [review]
Hi @GordonOtieno! Great job 👍 Cars are adding successfully and the form is looking nice 👏 Although, there are a few changes I suggested in the comments below please kindly effect the changes and then it'll be ready to merge. Well done!

File: src/components/addcar/AddCar.jsx (hunk 2 above)
Comment (GordonOtieno) [reply src/components/addcar/AddCar.jsx:117]: done

**Pull request merged**